.github/workflows/ci.yaml

```diff
@@ -18,36 +17,40 @@ jobs:
     steps:
     - uses: actions/checkout@v4
 
-    - name: Install uv
+    - name: Install UV
       uses: astral-sh/setup-uv@v5
 
     - uses: actions/setup-python@v5
       with:
         python-version: ${{ matrix.python-version }}
 
+    # Install the required dependencies (pytest, pytest-cov, requests, click)
+    - name: Install dependencies
+      run: |
+        pip install pytest pytest-cov requests click
```

Comment:
@Tatianah3 can you look at using the uvx command in the documentation to run this properly because then you don't need to to the pip install this way.

Please message me if you have questions